site/dat/docs/Reporting.md

```diff
@@ -152,4 +152,22 @@ modules:
     - 99.9
     - 100.0
 ```
-`rtimes-len` allows to reduce memory consumption for heavy tests. On the other hand, you reduce the precision of distribution with that. 
\ No newline at end of file
+`rtimes-len` allows to reduce memory consumption for heavy tests. On the other hand, you reduce the precision of distribution with that.
+ 
+ ## Pass/Fail Criteria Subsystem
+ 
+ Pass/Fail module is used to dynamically update test status based on some runtime criteria. For
+ example, you can use it to automatically fail the test when response time exceeds some threshold.
+ Here's a sample:
+ 
+ ```yaml
+ ---
+ reporting:
+ - module: passfail
+   criteria:
+   - avg-rt of IndexPage>150ms for 10s, stop as failed
+   - fail of CheckoutPage>50% for 10s, stop as failed
+ ```
+ 
+ You can learn more about Pass/Fail criteria capabilities at its [page](PassFail.md).
```

Comment:
In my opinion we should add next actual information into reporting.md -> JUnitXML:
1. 'data-source: passfail' is deprecated (and what replaces it)
2. order of junitXML and passfail matters.